compiling

this happens:
```
In file included from ./core/object.h:38,
                 from modules/qurobullet/bullet.h:4,
                 from modules/qurobullet/register_types.cpp:5:
./core/os/rw_lock.h:41:22: error: 'shared_timed_mutex' in namespace 'std' does not name a type
   41 |         mutable std::shared_timed_mutex mutex;
      |                      ^~~~~~~~~~~~~~~~~~
./core/os/rw_lock.h:41:17: note: 'std::shared_timed_mutex' is only available from C++14 onwards
   41 |         mutable std::shared_timed_mutex mutex;
      |                 ^~~
./core/os/rw_lock.h: In member function 'void RWLock::read_lock() const':
./core/os/rw_lock.h:46:17: error: 'mutex' was not declared in this scope; did you mean 'std::mutex'?
   46 |                 mutex.lock_shared();
      |                 ^~~~~
      |                 std::mutex
In file included from /usr/include/c++/11.2.0/mutex:43,
                 from ./core/os/mutex.h:39,
                 from ./core/string_name.h:34,
                 from ./core/node_path.h:34,
                 from ./core/hashfuncs.h:36,
                 from ./core/hash_map.h:35,
                 from ./core/object.h:34,
                 from modules/qurobullet/bullet.h:4,
                 from modules/qurobullet/register_types.cpp:5:
/usr/include/c++/11.2.0/bits/std_mutex.h:83:9: note: 'std::mutex' declared here
   83 |   class mutex : private __mutex_base
      |         ^~~~~
In file included from ./core/object.h:38,
                 from modules/qurobullet/bullet.h:4,
                 from modules/qurobullet/register_types.cpp:5:
./core/os/rw_lock.h: In member function 'void RWLock::read_unlock() const':
./core/os/rw_lock.h:51:17: error: 'mutex' was not declared in this scope; did you mean 'std::mutex'?
   51 |                 mutex.unlock_shared();
      |                 ^~~~~
      |                 std::mutex
In file included from /usr/include/c++/11.2.0/mutex:43,
                 from ./core/os/mutex.h:39,
                 from ./core/string_name.h:34,
                 from ./core/node_path.h:34,
                 from ./core/hashfuncs.h:36,
                 from ./core/hash_map.h:35,
                 from ./core/object.h:34,
                 from modules/qurobullet/bullet.h:4,
                 from modules/qurobullet/register_types.cpp:5:
/usr/include/c++/11.2.0/bits/std_mutex.h:83:9: note: 'std::mutex' declared here
   83 |   class mutex : private __mutex_base
      |         ^~~~~
In file included from ./core/object.h:38,
                 from modules/qurobullet/bullet.h:4,
                 from modules/qurobullet/register_types.cpp:5:
./core/os/rw_lock.h: In member function 'Error RWLock::read_try_lock() const':
./core/os/rw_lock.h:56:24: error: 'mutex' was not declared in this scope; did you mean 'std::mutex'?
   56 |                 return mutex.try_lock_shared() ? OK : ERR_BUSY;
      |                        ^~~~~
      |                        std::mutex
In file included from /usr/include/c++/11.2.0/mutex:43,
                 from ./core/os/mutex.h:39,
                 from ./core/string_name.h:34,
                 from ./core/node_path.h:34,
                 from ./core/hashfuncs.h:36,
                 from ./core/hash_map.h:35,
                 from ./core/object.h:34,
                 from modules/qurobullet/bullet.h:4,
                 from modules/qurobullet/register_types.cpp:5:
/usr/include/c++/11.2.0/bits/std_mutex.h:83:9: note: 'std::mutex' declared here
   83 |   class mutex : private __mutex_base
      |         ^~~~~
In file included from ./core/object.h:38,
                 from modules/qurobullet/bullet.h:4,
                 from modules/qurobullet/register_types.cpp:5:
./core/os/rw_lock.h: In member function 'void RWLock::write_lock()':
./core/os/rw_lock.h:61:17: error: 'mutex' was not declared in this scope; did you mean 'std::mutex'?
   61 |                 mutex.lock();
      |                 ^~~~~
      |                 std::mutex
In file included from /usr/include/c++/11.2.0/mutex:43,
                 from ./core/os/mutex.h:39,
                 from ./core/string_name.h:34,
                 from ./core/node_path.h:34,
                 from ./core/hashfuncs.h:36,
                 from ./core/hash_map.h:35,
                 from ./core/object.h:34,
                 from modules/qurobullet/bullet.h:4,
                 from modules/qurobullet/register_types.cpp:5:
/usr/include/c++/11.2.0/bits/std_mutex.h:83:9: note: 'std::mutex' declared here
   83 |   class mutex : private __mutex_base
      |         ^~~~~
In file included from ./core/object.h:38,
                 from modules/qurobullet/bullet.h:4,
                 from modules/qurobullet/register_types.cpp:5:
./core/os/rw_lock.h: In member function 'void RWLock::write_unlock()':
./core/os/rw_lock.h:66:17: error: 'mutex' was not declared in this scope; did you mean 'std::mutex'?
   66 |                 mutex.unlock();
      |                 ^~~~~
      |                 std::mutex
In file included from /usr/include/c++/11.2.0/mutex:43,
                 from ./core/os/mutex.h:39,
                 from ./core/string_name.h:34,
                 from ./core/node_path.h:34,
                 from ./core/hashfuncs.h:36,
                 from ./core/hash_map.h:35,
                 from ./core/object.h:34,
                 from modules/qurobullet/bullet.h:4,
                 from modules/qurobullet/register_types.cpp:5:
/usr/include/c++/11.2.0/bits/std_mutex.h:83:9: note: 'std::mutex' declared here
   83 |   class mutex : private __mutex_base
      |         ^~~~~
In file included from ./core/object.h:38,
                 from modules/qurobullet/bullet.h:4,
                 from modules/qurobullet/register_types.cpp:5:
./core/os/rw_lock.h: In member function 'Error RWLock::write_try_lock()':
./core/os/rw_lock.h:71:24: error: 'mutex' was not declared in this scope; did you mean 'std::mutex'?
   71 |                 return mutex.try_lock() ? OK : ERR_BUSY;
      |                        ^~~~~
      |                        std::mutex
In file included from /usr/include/c++/11.2.0/mutex:43,
                 from ./core/os/mutex.h:39,
                 from ./core/string_name.h:34,
                 from ./core/node_path.h:34,
                 from ./core/hashfuncs.h:36,
                 from ./core/hash_map.h:35,
                 from ./core/object.h:34,
                 from modules/qurobullet/bullet.h:4,
                 from modules/qurobullet/register_types.cpp:5:
/usr/include/c++/11.2.0/bits/std_mutex.h:83:9: note: 'std::mutex' declared here
   83 |   class mutex : private __mutex_base
      |         ^~~~~
In file included from ./core/pool_vector.h:36,
                 from ./core/math/transform_2d.h:35,
                 from ./core/method_ptrcall.h:34,
                 from ./core/method_bind.h:35,
                 from ./core/class_db.h:34,
                 from ./core/resource.h:34,
                 from modules/qurobullet/bullet_type.h:4,
                 from modules/qurobullet/bullet_type.cpp:1:
./core/os/rw_lock.h:41:22: error: 'shared_timed_mutex' in namespace 'std' does not name a type
   41 |         mutable std::shared_timed_mutex mutex;
      |                      ^~~~~~~~~~~~~~~~~~
./core/os/rw_lock.h:41:17: note: 'std::shared_timed_mutex' is only available from C++14 onwards
   41 |         mutable std::shared_timed_mutex mutex;
      |                 ^~~
./core/os/rw_lock.h: In member function 'void RWLock::read_lock() const':
./core/os/rw_lock.h:46:17: error: 'mutex' was not declared in this scope; did you mean 'std::mutex'?
   46 |                 mutex.lock_shared();
      |                 ^~~~~
      |                 std::mutex
In file included from /usr/include/c++/11.2.0/mutex:43,
                 from ./core/os/mutex.h:39,
                 from ./core/pool_vector.h:35,
                 from ./core/math/transform_2d.h:35,
                 from ./core/method_ptrcall.h:34,
                 from ./core/method_bind.h:35,
                 from ./core/class_db.h:34,
                 from ./core/resource.h:34,
                 from modules/qurobullet/bullet_type.h:4,
                 from modules/qurobullet/bullet_type.cpp:1:
/usr/include/c++/11.2.0/bits/std_mutex.h:83:9: note: 'std::mutex' declared here
   83 |   class mutex : private __mutex_base
      |         ^~~~~
In file included from ./core/pool_vector.h:36,
                 from ./core/math/transform_2d.h:35,
                 from ./core/method_ptrcall.h:34,
                 from ./core/method_bind.h:35,
                 from ./core/class_db.h:34,
                 from ./core/resource.h:34,
                 from modules/qurobullet/bullet_type.h:4,
                 from modules/qurobullet/bullet_type.cpp:1:
./core/os/rw_lock.h: In member function 'void RWLock::read_unlock() const':
./core/os/rw_lock.h:51:17: error: 'mutex' was not declared in this scope; did you mean 'std::mutex'?
   51 |                 mutex.unlock_shared();
      |                 ^~~~~
      |                 std::mutex
In file included from /usr/include/c++/11.2.0/mutex:43,
                 from ./core/os/mutex.h:39,
                 from ./core/pool_vector.h:35,
                 from ./core/math/transform_2d.h:35,
                 from ./core/method_ptrcall.h:34,
                 from ./core/method_bind.h:35,
                 from ./core/class_db.h:34,
                 from ./core/resource.h:34,
                 from modules/qurobullet/bullet_type.h:4,
                 from modules/qurobullet/bullet_type.cpp:1:
/usr/include/c++/11.2.0/bits/std_mutex.h:83:9: note: 'std::mutex' declared here
   83 |   class mutex : private __mutex_base
      |         ^~~~~
In file included from ./core/pool_vector.h:36,
                 from ./core/math/transform_2d.h:35,
                 from ./core/method_ptrcall.h:34,
                 from ./core/method_bind.h:35,
                 from ./core/class_db.h:34,
                 from ./core/resource.h:34,
                 from modules/qurobullet/bullet_type.h:4,
                 from modules/qurobullet/bullet_type.cpp:1:
./core/os/rw_lock.h: In member function 'Error RWLock::read_try_lock() const':
./core/os/rw_lock.h:56:24: error: 'mutex' was not declared in this scope; did you mean 'std::mutex'?
   56 |                 return mutex.try_lock_shared() ? OK : ERR_BUSY;
      |                        ^~~~~
      |                        std::mutex
In file included from /usr/include/c++/11.2.0/mutex:43,
                 from ./core/os/mutex.h:39,
                 from ./core/pool_vector.h:35,
                 from ./core/math/transform_2d.h:35,
                 from ./core/method_ptrcall.h:34,
                 from ./core/method_bind.h:35,
                 from ./core/class_db.h:34,
                 from ./core/resource.h:34,
                 from modules/qurobullet/bullet_type.h:4,
                 from modules/qurobullet/bullet_type.cpp:1:
/usr/include/c++/11.2.0/bits/std_mutex.h:83:9: note: 'std::mutex' declared here
   83 |   class mutex : private __mutex_base
      |         ^~~~~
In file included from ./core/pool_vector.h:36,
                 from ./core/math/transform_2d.h:35,
                 from ./core/method_ptrcall.h:34,
                 from ./core/method_bind.h:35,
                 from ./core/class_db.h:34,
                 from ./core/resource.h:34,
                 from modules/qurobullet/bullet_type.h:4,
                 from modules/qurobullet/bullet_type.cpp:1:
./core/os/rw_lock.h: In member function 'void RWLock::write_lock()':
./core/os/rw_lock.h:61:17: error: 'mutex' was not declared in this scope; did you mean 'std::mutex'?
   61 |                 mutex.lock();
      |                 ^~~~~
      |                 std::mutex
In file included from /usr/include/c++/11.2.0/mutex:43,
                 from ./core/os/mutex.h:39,
                 from ./core/pool_vector.h:35,
                 from ./core/math/transform_2d.h:35,
                 from ./core/method_ptrcall.h:34,
                 from ./core/method_bind.h:35,
                 from ./core/class_db.h:34,
                 from ./core/resource.h:34,
                 from modules/qurobullet/bullet_type.h:4,
                 from modules/qurobullet/bullet_type.cpp:1:
/usr/include/c++/11.2.0/bits/std_mutex.h:83:9: note: 'std::mutex' declared here
   83 |   class mutex : private __mutex_base
      |         ^~~~~
In file included from ./core/pool_vector.h:36,
                 from ./core/math/transform_2d.h:35,
                 from ./core/method_ptrcall.h:34,
                 from ./core/method_bind.h:35,
                 from ./core/class_db.h:34,
                 from ./core/resource.h:34,
                 from modules/qurobullet/bullet_type.h:4,
                 from modules/qurobullet/bullet_type.cpp:1:
./core/os/rw_lock.h: In member function 'void RWLock::write_unlock()':
./core/os/rw_lock.h:66:17: error: 'mutex' was not declared in this scope; did you mean 'std::mutex'?
   66 |                 mutex.unlock();
      |                 ^~~~~
      |                 std::mutex
In file included from /usr/include/c++/11.2.0/mutex:43,
                 from ./core/os/mutex.h:39,
                 from ./core/pool_vector.h:35,
                 from ./core/math/transform_2d.h:35,
                 from ./core/method_ptrcall.h:34,
                 from ./core/method_bind.h:35,
                 from ./core/class_db.h:34,
                 from ./core/resource.h:34,
                 from modules/qurobullet/bullet_type.h:4,
                 from modules/qurobullet/bullet_type.cpp:1:
/usr/include/c++/11.2.0/bits/std_mutex.h:83:9: note: 'std::mutex' declared here
   83 |   class mutex : private __mutex_base
      |         ^~~~~
In file included from ./core/pool_vector.h:36,
                 from ./core/math/transform_2d.h:35,
                 from ./core/method_ptrcall.h:34,
                 from ./core/method_bind.h:35,
                 from ./core/class_db.h:34,
                 from ./core/resource.h:34,
                 from modules/qurobullet/bullet_type.h:4,
                 from modules/qurobullet/bullet_type.cpp:1:
./core/os/rw_lock.h: In member function 'Error RWLock::write_try_lock()':
./core/os/rw_lock.h:71:24: error: 'mutex' was not declared in this scope; did you mean 'std::mutex'?
   71 |                 return mutex.try_lock() ? OK : ERR_BUSY;
      |                        ^~~~~
      |                        std::mutex
In file included from /usr/include/c++/11.2.0/mutex:43,
                 from ./core/os/mutex.h:39,
                 from ./core/pool_vector.h:35,
                 from ./core/math/transform_2d.h:35,
                 from ./core/method_ptrcall.h:34,
                 from ./core/method_bind.h:35,
                 from ./core/class_db.h:34,
                 from ./core/resource.h:34,
                 from modules/qurobullet/bullet_type.h:4,
                 from modules/qurobullet/bullet_type.cpp:1:
/usr/include/c++/11.2.0/bits/std_mutex.h:83:9: note: 'std::mutex' declared here
   83 |   class mutex : private __mutex_base
      |         ^~~~~
[Initial build] Compiling ==> modules/qurobullet/bullet.cpp
In file included from ./core/object.h:38,
                 from modules/qurobullet/bullet.h:4,
                 from modules/qurobullet/bullet.cpp:1:
./core/os/rw_lock.h:41:22: error: 'shared_timed_mutex' in namespace 'std' does not name a type
   41 |         mutable std::shared_timed_mutex mutex;
      |                      ^~~~~~~~~~~~~~~~~~
./core/os/rw_lock.h:41:17: note: 'std::shared_timed_mutex' is only available from C++14 onwards
   41 |         mutable std::shared_timed_mutex mutex;
      |                 ^~~
./core/os/rw_lock.h: In member function 'void RWLock::read_lock() const':
./core/os/rw_lock.h:46:17: error: 'mutex' was not declared in this scope; did you mean 'std::mutex'?
   46 |                 mutex.lock_shared();
      |                 ^~~~~
      |                 std::mutex
In file included from /usr/include/c++/11.2.0/mutex:43,
                 from ./core/os/mutex.h:39,
                 from ./core/string_name.h:34,
                 from ./core/node_path.h:34,
                 from ./core/hashfuncs.h:36,
                 from ./core/hash_map.h:35,
                 from ./core/object.h:34,
                 from modules/qurobullet/bullet.h:4,
                 from modules/qurobullet/bullet.cpp:1:
/usr/include/c++/11.2.0/bits/std_mutex.h:83:9: note: 'std::mutex' declared here
   83 |   class mutex : private __mutex_base
      |         ^~~~~
In file included from ./core/object.h:38,
                 from modules/qurobullet/bullet.h:4,
                 from modules/qurobullet/bullet.cpp:1:
./core/os/rw_lock.h: In member function 'void RWLock::read_unlock() const':
./core/os/rw_lock.h:51:17: error: 'mutex' was not declared in this scope; did you mean 'std::mutex'?
   51 |                 mutex.unlock_shared();
      |                 ^~~~~
      |                 std::mutex
In file included from /usr/include/c++/11.2.0/mutex:43,
                 from ./core/os/mutex.h:39,
                 from ./core/string_name.h:34,
                 from ./core/node_path.h:34,
                 from ./core/hashfuncs.h:36,
                 from ./core/hash_map.h:35,
                 from ./core/object.h:34,
                 from modules/qurobullet/bullet.h:4,
                 from modules/qurobullet/bullet.cpp:1:
/usr/include/c++/11.2.0/bits/std_mutex.h:83:9: note: 'std::mutex' declared here
   83 |   class mutex : private __mutex_base
      |         ^~~~~
In file included from ./core/object.h:38,
                 from modules/qurobullet/bullet.h:4,
                 from modules/qurobullet/bullet.cpp:1:
./core/os/rw_lock.h: In member function 'Error RWLock::read_try_lock() const':
./core/os/rw_lock.h:56:24: error: 'mutex' was not declared in this scope; did you mean 'std::mutex'?
   56 |                 return mutex.try_lock_shared() ? OK : ERR_BUSY;
      |                        ^~~~~
      |                        std::mutex
In file included from /usr/include/c++/11.2.0/mutex:43,
                 from ./core/os/mutex.h:39,
                 from ./core/string_name.h:34,
                 from ./core/node_path.h:34,
                 from ./core/hashfuncs.h:36,
                 from ./core/hash_map.h:35,
                 from ./core/object.h:34,
                 from modules/qurobullet/bullet.h:4,
                 from modules/qurobullet/bullet.cpp:1:
/usr/include/c++/11.2.0/bits/std_mutex.h:83:9: note: 'std::mutex' declared here
   83 |   class mutex : private __mutex_base
      |         ^~~~~
In file included from ./core/object.h:38,
                 from modules/qurobullet/bullet.h:4,
                 from modules/qurobullet/bullet.cpp:1:
./core/os/rw_lock.h: In member function 'void RWLock::write_lock()':
./core/os/rw_lock.h:61:17: error: 'mutex' was not declared in this scope; did you mean 'std::mutex'?
   61 |                 mutex.lock();
      |                 ^~~~~
      |                 std::mutex
In file included from /usr/include/c++/11.2.0/mutex:43,
                 from ./core/os/mutex.h:39,
                 from ./core/string_name.h:34,
                 from ./core/node_path.h:34,
                 from ./core/hashfuncs.h:36,
                 from ./core/hash_map.h:35,
                 from ./core/object.h:34,
                 from modules/qurobullet/bullet.h:4,
                 from modules/qurobullet/bullet.cpp:1:
/usr/include/c++/11.2.0/bits/std_mutex.h:83:9: note: 'std::mutex' declared here
   83 |   class mutex : private __mutex_base
      |         ^~~~~
In file included from ./core/object.h:38,
                 from modules/qurobullet/bullet.h:4,
                 from modules/qurobullet/bullet.cpp:1:
./core/os/rw_lock.h: In member function 'void RWLock::write_unlock()':
./core/os/rw_lock.h:66:17: error: 'mutex' was not declared in this scope; did you mean 'std::mutex'?
   66 |                 mutex.unlock();
      |                 ^~~~~
      |                 std::mutex
In file included from /usr/include/c++/11.2.0/mutex:43,
                 from ./core/os/mutex.h:39,
                 from ./core/string_name.h:34,
                 from ./core/node_path.h:34,
                 from ./core/hashfuncs.h:36,
                 from ./core/hash_map.h:35,
                 from ./core/object.h:34,
                 from modules/qurobullet/bullet.h:4,
                 from modules/qurobullet/bullet.cpp:1:
/usr/include/c++/11.2.0/bits/std_mutex.h:83:9: note: 'std::mutex' declared here
   83 |   class mutex : private __mutex_base
      |         ^~~~~
In file included from ./core/object.h:38,
                 from modules/qurobullet/bullet.h:4,
                 from modules/qurobullet/bullet.cpp:1:
./core/os/rw_lock.h: In member function 'Error RWLock::write_try_lock()':
./core/os/rw_lock.h:71:24: error: 'mutex' was not declared in this scope; did you mean 'std::mutex'?
   71 |                 return mutex.try_lock() ? OK : ERR_BUSY;
      |                        ^~~~~
      |                        std::mutex
In file included from /usr/include/c++/11.2.0/mutex:43,
                 from ./core/os/mutex.h:39,
                 from ./core/string_name.h:34,
                 from ./core/node_path.h:34,
                 from ./core/hashfuncs.h:36,
                 from ./core/hash_map.h:35,
                 from ./core/object.h:34,
                 from modules/qurobullet/bullet.h:4,
                 from modules/qurobullet/bullet.cpp:1:
/usr/include/c++/11.2.0/bits/std_mutex.h:83:9: note: 'std::mutex' declared here
   83 |   class mutex : private __mutex_base
      |         ^~~~~
scons: *** [modules/qurobullet/register_types.x11.opt.tools.64.o] Error 1
scons: *** [modules/qurobullet/bullet_type.x11.opt.tools.64.o] Error 1
scons: *** [modules/qurobullet/bullet.x11.opt.tools.64.o] Error 1
scons: building terminated because of errors.
[Time elapsed: 00:06:08.359]
```
when compiling with: 
```
scons target=release_debug tools=yes module_bmp_enabled=no module_bullet_enabled=no module_csg_enabled=no module_dds_enabled=no module_enet_enabled=no module_etc_enabled=no module_gridmap_enabled=no module_hdr_enabled=no module_mbedtls_enabled=no module_mobile_vr_enabled=no module_opus_enabled=no module_pvr_enabled=no module_recast_enabled=no module_webm_enabled=no optimize=speed use_lto=yes -j6
```
what module am i missing?

--- a/SCsub
+++ b/SCsub
@@ -13,7 +13,6 @@ sources = [
 
 module_env = env.Clone()
 module_env.Append(CCFLAGS=['-O2'])
-module_env.Append(CXXFLAGS=['-std=c++11'])
 
 if ARGUMENTS.get('qurobullet_shared', 'no') == 'yes':
     # Shared lib compilation

